FootprintIOS/FootprintIOS/Sources/Config/Enviroment.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Enviroment.swift
+//  Footprint-iOS
+//
+//  Created by 송영모 on 2022/08/31.
+//  Copyright © 2022 Footprint-iOS. All rights reserved.
+//
+
+import Foundation
+
+class Enviroment {
```

Comment:
사소하지만 .. 오타가 난 것 같아서 말씀드려요 !
```suggestion
class Environment {
```